elasticsearch/run.sh

```diff
@@ -151,9 +151,13 @@ exit 1
 
 seed_searchguard(){
     info Seeding the searchguard ACL index
+
+    config=/usr/share/elasticsearch/config/elasticsearch.yml
```

Comment:
probably... i feel like there is a bunch of settings that should probably be moved to some sort of ENV script